payment-service/src/main/java/com/hedvig/paymentservice/query/adyenAccount/AdyenAccountRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.hedvig.paymentservice.query.adyenAccount
+
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+
+
+@Repository
+interface AdyenAccountRepository : CrudRepository<AdyenAccount, String>
```

Comment:
Don't remember exact reason but I think Jpa has a bit more power in it 🤔 
```suggestion
interface AdyenAccountRepository : JpaRepository<AdyenAccount, String>
```